test/unity/resource/test_host.py

```diff
@@ -105,6 +111,96 @@ def test_delete_host_and_ip_port(self):
         assert_that(resp.is_ok(), equal_to(True))
         assert_that(ip_port.delete, raises(UnityResourceNotFoundError))
 
+    @patch_rest()
+    def test_add_initiator(self):
+        host = UnityHost(cli=t_rest(), _id='Host_9')
+        wwn = "50:00:14:40:47:B0:0C:44:50:00:14:42:D0:0C:44:10"
+        initiator = host.add_initiator(wwn, HostInitiatorTypeEnum.FC)
```

Comment:
As I can see, the type parameter is redundant.  Please default it and analyze the uid to set the default value.